### DISCUSSION TITLE
Create users without sending an email

### DISCUSSION THREAD

**Discussion by prostheboss** [Leantime/leantime#1258] [Q&A]
I am currently unable to send emails for user signup, is there any way to manually create a user and allow them to change their password later, or perhaps give a token to sign up with?

**Comment by wa3ziz** [comment]
I am facing the same issue. 
Is there a way to assign a default password to newly created user and force password change upon connection?

**Reply by prostheboss** [reply]
I ended up modifying the SQL database and copied my main passwords hash, then logged in and changed it manually, I also had to change another value, I can't remember it off the top of my head but it was something to do with being active or not

**Comment by marcelfolaron** [comment]
As of 2.3.6 you can get the invite link off of the user details page.
